README.md

```diff
@@ -47,9 +47,11 @@ The *ArcGIS Maps SDK for Swift Toolkit* has a *Target SDK* version of *16.0*, me
 
  New to Swift Package Manager? Visit [swift.org/package-manager/](https://swift.org/package-manager/).
 
-## Configure API Key
+## Configure API Key & Licensing
 
-Some of the toolkit components and examples utilize a set of ready-to-use ArcGIS Platform services, including basemaps, and therefore require an API Key to be set in `ExamplesApp.swift`. Please see the [setup guide](https://developers.arcgis.com/swift/get-started/#3-access-services-and-content-with-an-api-key/) for more information.
+Use of ArcGIS location services, such as basemaps, geocoding, and routing, requires either user authentication or API key authentication. Some of the toolkit components and examples utilize a set of these ready-to-use ArcGIS location services, including basemaps, and therefore require an API Key to be set in `ExamplesApp.swift`. Please see the [setup guide](https://developers.arcgis.com/swift/get-started/#3-access-services-and-content-with-an-api-key/), https://links.esri.com/arcgis-maps-sdk-security-auth and https://links.esri.com/arcgis-maps-sdk-license-and-deploy for more information.
```

Comment:
1. This is a slight modification to the suggested text to include the pre-existing "Some of the toolkit components..." language.
2. The 2nd and 3rd links can be updated to display a title `[]()` once those webpages are live and we have a good title to use.
3. This assumes the cutoff suggestion meant to include the 2nd link in the referenced issue (https://links.esri.com/arcgis-maps-sdk-license-and-deploy).

Reply:
"such as basemap styles, geocoding, and routing services,"

Reply:
"including basemap styles"

Reply:
How about.... "Please see the [Get started guide](https://developers.arcgis.com/swift/get-started/#3-get-an-access-token), [Create an API Key](https://developers.arcgis.com/documentation/security-and-authentication/api-key-authentication/tutorials/create-an-api-key/) and [API Key Authentication](https://main.sites.afd.arcgis.com/swift/security-and-authentication/#api-key-authentication)"